metadb/config.go

```diff
@@ -56,6 +59,9 @@ func (config *Config) Validate() error {
 	if config.DataDir == "" {
 		return fmt.Errorf("%s: %s", errValidationStr, "data_dir")
 	}
+	if config.BlockInterval == 0 {
+		config.BlockInterval = 48 // set default
```

Comment:
should set default config in `func NewConfig() *Config`.
Code here to validate, not set value.